packages/@okta/vuepress-site/docs/guides/add-an-external-idp/create-an-app-at-idp/openidconnect/appatidp.md

```diff
@@ -1,4 +1,4 @@
-Use the Identity Provider's documentation to create a client application. For example, if you want to use PayPal as your OpenID Connect Identity Provider, click **Create App** on [this page](https://developer.paypal.com/developer/applications/) to create a PayPal client application.
+Use the Identity Provider's documentation to create a client application. 
```

Comment:
I talked to Venkat about this, because I didn't want to leave it with no example, so we agreed on referring to their existing Identity Provider. Here's my suggested change:

```suggestion
Use the Identity Provider's documentation to create a client application, for example, when you want to use your existing Identity Provider as your OpenID Connect Identity Provider.```